gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.caching=true
```

Comment:
What's the benefit of this? I'd assume it uses the local build cache out-of-the-box?

Reply:
Exactly